problem2.py

```diff
@@ -1 +1,33 @@
 
+"""
+-------------------------------------------------------------------------------
+Name:   whileloop.py
+Purpose:  find the distance driven per day
+ 
+Author: Chan.E
+ 
+Created:  03/03/2021
+------------------------------------------------------------------------------
+"""
+
+print ("****Welcome to the DoorDash Distance Tracker****")
+
+#find the distance travelled
+print ("**Travel Log***")
+total = 0
+dis_travel = 0
+
+kilometers = int(input("Enter distance travelled for the day (100 to stop): "))
```

Comment:
ideally there should only be one prompt for km input, and that should be in the loop